packages/terra-search-field/tests/jest/SearchField.test.jsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import SearchField from '../../src/SearchField';
+
+
+describe('Snapshots', () => {
+  it('renders a basic search field', () => {
+    const searchField = shallow(<SearchField />);
+    expect(searchField).toMatchSnapshot();
+  });
+
+  it('renders a search field with a placeholder', () => {
+    const searchField = shallow(<SearchField placeholder="Test" />);
+    expect(searchField).toMatchSnapshot();
+  });
+
+  it('renders a search field with a value', () => {
+    const searchField = shallow(<SearchField />);
+    searchField.setState({ searchText: 'Test' });
+
+    expect(searchField).toMatchSnapshot();
+  });
+});
+
+describe('Manual Search', () => {
```

Comment:
Search will occur on input after the provided delay. It is does not currently trigger a search on "enter" press.